### ISSUE TITLE
Running remove-default-apps.ps1 causes the windows store not to open

### ISSUE THREAD

**Issue by darthcircuit** [W4RH4WK/Debloat-Windows-10#45]
I'm preparing an image to use with MDT/SCCM. I've used your script to clean up the provisioned apps as well local user apps. I commented out the Store in the list. While windows doesn't seem to uninstall it, the app will no longer open or update any other apps (causing them to not open either. Calculator, Groove Music, etc.) - The apps it does remove are removed completely from the start menu.

I'm on build 1511. I only made a few changes to your script to make it work from my batch which kickstarts it. (I copied the takeown module into the local directory)

This is what it looks like with my mods:
```

#   Description:
# This script removes unwanted Apps that come with Windows. If you  do not want
# to remove certain Apps comment out the corresponding lines below.

ls -Recurse *.psm1 | Unblock-File
ls -Recurse *.ps1 | Unblock-File

Import-Module -DisableNameChecking .\take-own.psm1

echo "Elevating priviledges for this process"
do {} until (Elevate-Privileges SeTakeOwnershipPrivilege)

echo "Uninstalling default apps"
$apps = @(
    # default Windows 10 apps
    "Microsoft.3DBuilder"
    "Microsoft.Appconnector"
    "Microsoft.BingFinance"
    "Microsoft.BingNews"
    "Microsoft.BingSports"
    "Microsoft.BingWeather"
    "Microsoft.Getstarted"
    "Microsoft.MicrosoftOfficeHub"
    "Microsoft.MicrosoftSolitaireCollection"
    "Microsoft.Office.OneNote"
    "Microsoft.People"
    "Microsoft.SkypeApp"
    #"Microsoft.Windows.Photos"
    "Microsoft.WindowsAlarms"
    #"Microsoft.WindowsCalculator"
    "Microsoft.WindowsCamera"
    "Microsoft.WindowsMaps"
    "Microsoft.WindowsPhone"
    "Microsoft.WindowsSoundRecorder"
    #"Microsoft.WindowsStore"
    "Microsoft.XboxApp"
    #"Microsoft.ZuneMusic"
    "Microsoft.ZuneVideo"
    "microsoft.windowscommunicationsapps"
    "Microsoft.MinecraftUWP"

    # Threshold 2 apps
    "Microsoft.CommsPhone"
    "Microsoft.ConnectivityStore"
    "Microsoft.Messaging"
    "Microsoft.Office.Sway"

    # non-Microsoft
    "9E2F88E3.Twitter"
    "Flipboard.Flipboard"
    "ShazamEntertainmentLtd.Shazam"
    "king.com.CandyCrushSaga"
    "ClearChannelRadioDigital.iHeartRadio"

    # apps which cannot be removed using Remove-AppxPackage
    #"Microsoft.BioEnrollment"
    #"Microsoft.MicrosoftEdge"
    #"Microsoft.Windows.Cortana"
    #"Microsoft.WindowsFeedback"
    #"Microsoft.XboxGameCallableUI"
    #"Microsoft.XboxIdentityProvider"
    #"Windows.ContactSupport"
)

foreach ($app in $apps) {
    Get-AppxPackage -Name $app -AllUsers | Remove-AppxPackage

    Get-AppXProvisionedPackage -Online |
        where DisplayName -EQ $app |
        Remove-AppxProvisionedPackage -Online
}
```

**Comment by W4RH4WK** [comment]
Seems like a new features :p,  just kidding. I'll look into this, maybe the App Store has a silent dependency on one of the Apps.

**Comment by W4RH4WK** [comment]
I just ran your script (changing only the import lines) and it seems to work fine on my end. I made sure to have the latest updates installed and rebooted after running the script. Can you provide additional information on your setup. Are you using the Home edition of Windows 10?

![Screenshot](http://imgur.com/P12nppRl.png)

**Comment by darthcircuit** [comment]
I replied via email this morning, but i guess that didn't go through.

I am running Enterprise Windows 10 - 1511.

I'm deploying Windows with MDT and ran your script before sysprepping and capturing the WIM.

I did some more testing this morning, and I disabled a different script I ran prior to yours that cleaned the Windows Update cache. I guess both scripts don't play well together.

Either way, its working for me now. Thanks for looking into it for me :)
